src/.formatted-files

```diff
@@ -0,0 +1,8 @@
+stat.h
+tweak.cpp
+tweak.h
+thinblock.cpp
+thinblock.h
+leakybucket.h
+parallel.cpp
+parallel.h
```

Comment:
possibly.  The advantage is that we are checking the end state that we want -- just like how we run all the regression tests even if the changes don't affect some subfunction of the code.  The disadvantage is that if the formatting in one of these files is bad every subsequent PR will fail until its fixed.  But we won't refuse a simple formatting fix, right?!  We'll actually follow our rules. ;-)